### PR TITLE
Feature: Add Speaker Notes for Polish Slides

### PR DESCRIPTION
**Pull request by christineberger**

### CODE REVIEW

File: _lessons/polish-project.html [hunk 1]
```diff
@@ -50,51 +68,126 @@
...
+""  
+  
+[Teacher Notes:]  
+You can take a moment here to exemplify what happens when you add this config.  
+You should hit these points:  
+- How does a dbt run function when your version is within the requirements?
+- How does a dbt run function when your version is out of the requirements?
+  
+You'll need to click on the status in the bottom right corner and scroll all  
+the way down the error message to see the message where the version doesn't
```
Comment (coapacetic) [inline _lessons/polish-project.html:92]: suggestion: we don't _always_ make this super clear, would this be worth escalating into a demo??

**Review by coapacetic — APPROVED** [review]
Thanks for documenting all these speaker notes @christineberger! I made one suggestion for an opportunity for an explicit demo.  Otherwise LGTM, yielding to @alexiswo for a second review.

**Comment by christineberger** [comment]
@coapacetic @alexiswo Let me know when this one is good to go! I made one change, which was to make an explicit demo from the require-version slide.

**Review by alexiswo — APPROVED** [review]
This is wonderful Christine! It will make it so much easier for new and experienced instructors to teach this section. lgtm! Feel free to merge it in

**Pull request merged**